lib/Dialect/LinalgExt/Transforms/LinalgExtBufferization.cpp

```diff
@@ -341,9 +342,11 @@ struct ParallelInsertSliceOpInterface
 
 void mlir::iree_compiler::IREE::LinalgExt::
     registerBufferizableOpInterfaceExternalModels(DialectRegistry &registry) {
-  registry.addOpInterface<InParallelOp, InParallelOpInterface>();
-  registry
-      .addOpInterface<PerformConcurrentlyOp, PerformConcurrentlyOpInterface>();
-  registry
-      .addOpInterface<ParallelInsertSliceOp, ParallelInsertSliceOpInterface>();
+  registry.addExtension(+[](MLIRContext *ctx, IREELinalgExtDialect *dialect) {
```

Comment:
Same Q here, I do not know this `(+[]` form ?

Reply:
Ah nm, found it: https://stackoverflow.com/questions/18889028/a-positive-lambda-what-sorcery-is-this

Reply:
also saw this for the first time yesterday...